src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -40,7 +30,16 @@
 					</option>
 				</kv-select>
 			</div>
-			<div v-if="!removeGenderDropdown" class="tw-flex tw-flex-col tw-grow">
+			<div v-if="!removeGenderDropdown && !withCategories" class="tw-overflow-scroll tw-pb-1">
```

Comment:
This should probably be `tw-overflow-x-auto`. The box I'm seeing in the screenshot in another comment is the scrollbars always being visible.